docs/source/installation/for_dev.rst

```diff
@@ -12,8 +12,8 @@ steps to clone your fork.
 
    If you have installed a non-developer version of manim, please uninstall
    it. This is to avoid any accidental usage of the non-developer version
-   when developing and testing on your local copy of the repository. This
```

Comment:
this made more sense imo.

Reply:
Again it’s a preference but this is grammatically correct

Reply:
when working on your local copy and when testing your local copy...

is 'develop' like work? do you develop on a local copy? I'm not sure.

Reply:
'develop' here means you working on the projects, usually people who aren't developer download a copy without GitHub and use it. So it is like a standalone software with no updates. But for developers we need the newest (nightly) version.